pages/entropy/generate-random-numbers/evm.mdx

```diff
@@ -58,6 +58,55 @@ contract YourContract is IEntropyConsumer {
 
 ## Usage
 
+### requestV2 Variants
```

Comment:
Do you think we should mention these variants in this guide?
I wanted these guides short. Reading time for these guides shouldn't exceed 3-4 minutes.

Maybe, you can add these variants on another page.